### DISCUSSION TITLE
Profile vs Centroid spectra

### DISCUSSION THREAD

**Discussion by LKittinun** [vdemichev/DiaNN#1261] [General]
Hi Vadim, thank you for such fantastic software. I would like to ask quick questions:
- Is there a benefit of using profile vs centroid data in Diann?
- Does Diann incorporate MS1 into the quantification? Will enhancing the MS1 method, such as BoxCar, improve the analysis?

Best,
Pete

**Comment by vdemichev** [comment]
Hi Pete,

- The data must be centroid, if you are referring to mzML. 
- With QuantUMS - yes. Not sure if BoxCar can be beneficial here, never tested.

Best,
Vadim

**Reply by LKittinun** [reply]
Dear Vadim

1. No, I meant specifically meant for .raw data. Will there be any improvement?
2. Nice, thank you so much.

Best,
Pete

**Reply by vdemichev** [reply]
Yes, DIA-NN only needs centroided.

**Reply by LKittinun** [reply]
Thank you very much Vadim. Have a nice day!
